EXPLAINER.md

```diff
@@ -162,13 +162,11 @@ const abortController = new AbortController();
 abortController.signal.onabort = event => {
   // All NFC operations have been aborted.
 };
- 
-const reader = new NDEFReader();
-await reader.scan({ signal: abortController.signal });
- 
-const writer = new NDEFWriter();
-await writer.push("foo", { signal: abortController.signal });
- 
+
+await navigator.nfc.ndef.scan({ signal: abortController.signal });
```

Comment:
I'd remove those two `await` otherwise `document.querySelector("#abortButton").onclick` declaration wouldn't be valid.